habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/CacheWheelsMojo.java

```diff
@@ -0,0 +1,54 @@
+package org.technologybrewery.habushu;
+
+import java.io.File;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.technologybrewery.habushu.exec.PoetryCommandHelper;
+import org.technologybrewery.habushu.util.HabushuUtil;
+
+/**
+ * Caches the generated poetry Wheel file when the
+ * {@link cacheWheels} flag configuration is set to 
+ * true during the {@link LifecyclePhase#INSTALL} 
+ * build phase.
+ * By default, the {@link cacheWheels} flag is set 
+ * to false and will not cache the wheel files. 
+ */
+@Mojo(name = "cache-wheels", defaultPhase = LifecyclePhase.INSTALL)
+public class CacheWheelsMojo extends AbstractHabushuMojo {
+
+    @Parameter(property = "habushu.cacheWheels", required = false, defaultValue = "false")
+    protected boolean cacheWheels;
```

Comment:
S: did we capture this config in the project readme?  Not seeing it in this commit, but perhaps it was in the prior commit?  Wondering if we would want to clarify when it is done as well (e.g., part of the install lifecycle)?

Reply:
To be more specific w/ my feedback, I see that it is in the general configuration docs.  But I think we should add it to the [lifecycle part of the docs](https://github.com/TechnologyBrewery/habushu?tab=readme-ov-file#the-habushu-build-lifecycle) too.

Reply:
@d-ryan-ashcraft Here's my draft

```
**install**
Publishes the pom.xml for the module into your local Maven Repository (~/.m2/repository). If the cacheWheels configuration is set to True, the wheel archive will be copied to the poetry cache directory (~/{poetry-cache-dir}/cache/repositories/wheels/{artifact-id}/). The cacheWheels configuration default behavior is not to cache the wheel archive.
```